wrappers/desktop.go

```diff
@@ -119,6 +120,17 @@ func rewriteExecLine(s *snap.Info, desktopFile, line string) (string, error) {
 		}
 	}
 
+	logger.Noticef("cannot use line %q for desktop file %q (snap %s)", line, desktopFile, s.Name())
```

Comment:
shouldn't this also mention that we will use a fallback?

Reply:
adding `; using %q instead` to the end of that message might be good indeed